README.md

```diff
@@ -1 +1,64 @@
-# toDoList
\ No newline at end of file
+![https://www.microverse.org/](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TODO App
+
+> App where the user can add books to a collection
+
+## :red_circle: Live Demo:
+
+[https://natajenkins.github.io/toDoList/]()
+
+## :hammer: Built with
+
+- HTML5
+- CSS3
+- JavaScript
+- Linters
+- VS code
+- Webpack
+
+## :construction_worker: Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/pointerish/Books/).
```

Comment:
Kindly add the test instruction in the README file(That is; how to run the test)